$OpenBSD: patch-setup_py,v 1.3 2019/04/25 14:26:43 danj Exp $

Disable blackd as we don't have a port for aiohttp-cors

Index: setup.py
--- setup.py.orig
+++ setup.py
@@ -60,7 +60,6 @@ setup(
     entry_points={
         "console_scripts": [
             "black=black:patched_main",
-            "blackd=blackd:patched_main [d]",
         ]
     },
 )
